src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3054,6 +3067,21 @@ void addProperties(ImmutableArray<ParameterSymbol> recordParameters)
                         builder.InstanceInitializersForRecordDeclarationWithParameters.Insert(addedCount, new FieldOrPropertyInitializer.Builder(property.BackingField, paramList.Parameters[param.Ordinal]));
                         addedCount++;
                     }
+                    else if (existingMember is PropertySymbol { IsStatic: false, GetMethod: { } } prop
```

Comment:
Since we're checking for an explicit `GetMethod`, consider testing with base property with `set` only.

Reply:
A setter-only property is tested in `Inheritance_18`. I can add another test which puts the setter-only property in a base class.